src/main/java/jalau/at18/katas/pokerhands/mauricio/TwoPairs.java

```diff
@@ -0,0 +1,25 @@
+package jalau.at18.katas.pokerhands.mauricio;
+
+public class TwoPairs extends CardValuesCounter {
+    private final int one = 1;
+    private final int two = 2;
+    private CardValue value1;
+    private CardValue value2;
+    private CardValue value3;
```

Comment:
If possible, try to refactor so we don't break the 2 attributes max per class limit